tests/pytorch/nightly/cpp-ops.libsonnet

```diff
@@ -54,7 +68,7 @@ local utils = import 'templates/utils.libsonnet';
   },
 
   configs: [
-    operations + v2_8 + common.Functional + timeouts.Hours(4),
+    operations + v2_8 + common.Functional + timeouts.Hours(4) + tpuVm,
```

Comment:
according to the below comments, there are some different way to enable TPUVM, we might need to check with this bug and author for more details

Reply:
checked and deprecated the commented used tpuvm,
decide to use the `tpuVm` like other tests